### PR TITLE
deps: bump bpmn-js from 8.10.0 to 9.2.2

### PR DESCRIPTION
**Pull request by boris-petrov**
cc @pinussilvestrus

### CODE REVIEW

File: package-lock.json [hunk 1]
```diff
@@ -1,8 +1,8692 @@
 {
   "name": "bpmn-js-executable-fix",
   "version": "0.1.3",
-  "lockfileVersion": 1,
+  "lockfileVersion": 2,
```
Comment (marstamm) [inline package-lock.json:4]: Please keep `lockfileVersion: 1`, as our downstream libraries and CI use old LTS Node versions

**Review by marstamm — CHANGES_REQUESTED** [review]
Thank you for your Contribution. Just some small comment from my side 🙂

File: package-lock.json (hunk 1 above)
Comment (boris-petrov) [reply package-lock.json:4]: hm, any ideas how I can do that on the latest NPM version?

Comment (marstamm) [reply package-lock.json:4]: I don't think it is possible with the latest version. I'm using nvm to switch to `node 12`. If it is okay with you, I can also bump the versions and close your PRs

**Comment by marstamm** [comment]
Not sure if I can update this PR, as it is in your repo. I created a new PR here: https://github.com/bpmn-io/bpmn-js-executable-fix/pull/30

Can you give it a quick check if it works for you?

**Comment by boris-petrov** [comment]
Yes, look great, thanks! The important thing is the `peerDependencies` update so I'm happy. :)

**Comment by boris-petrov** [comment]
Thank you too! :)

**Pull request closed without merging**